### ISSUE TITLE
What is the minimum GPU memory required to train LResNet100-E-IR on setting MS1M@ArcFace?

### ISSUE THREAD

**Issue by Jumabek** [deepinsight/insightface#124]
@nttstar 
1. I am new to Face Recognition and I have a limited hardware resources.
Could you approximately tell us the minimum GPU mem required to train `LResNet100-E-IR` on setting `MS1M@ArcFace`? 
For batch_size of 1 and 128 would be perfect.
I currently have 2 GTX 1080s

2. Could you tell us the approximate performance if I were to train `MS1M@Triplet` because in case my resources do not suffice  for Softmax/ArcFace Traing I will go with `MS1M@Triplet`

Thank you very much!

**Comment by Jumabek** [comment]
Okay, I found your answer here "For resnet100 with per-batch-size=128, you need about 18G GPU memory on each device."

 https://github.com/deepinsight/insightface/issues/86#issuecomment-372030300

**Comment by Jumabek** [comment]
How about my second question though?

**Comment by nttstar** [comment]
I'm not quite sure about the performance of triplet loss if you try to train it from scratch.

**Comment by huyangc** [comment]
What about the speed of training tripletloss? I'm running at 14.45 samples/secs with 8GPUs
@nttstar

**Comment by nttstar** [comment]
Tripletloss has extra hard-mining step so speed can not be told simply.
